### PR TITLE
Uprade storybook and webpack

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:** 

**Description of changes:**
- Fixes few security vulnerabilities.
- Upgrade storybook's webpack dependency to v5.
- Fixes few external links with markdown syntax
  that were fixed in Storybook alpha versions.
- Fix table rendering in migration v3 doc.
- UI component unit tests were failing due to `@testing-library/dom` module not found, I had to install it to make it work. No root cause on it yet.

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
Running `npm start` locally to start the storybook and check if loads fine.

3. If you made changes to the component library, have you provided corresponding documentation changes?
NA

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/components/migrationToV3.stories.mdx [hunk 1]
```diff
@@ -539,11 +539,17 @@ In V3, we have improved the `type` for selected device in `MeetingManager` and c
 
 #### MeetingManager Properties Change
 
-| Before in 2.x                              | After in 3.x                                             |
```
Comment (devalevenkatesh) [inline src/components/migrationToV3.stories.mdx:542]: This was rendered differently in storybook due to the `|` coming before the table separator:
Screen Shot 2022-04-19 at 3 46 34 PM

File: src/components/migrationToV3.stories.mdx [hunk 2]
```diff
@@ -27,7 +27,7 @@ Amazon Chime SDK React Components Library v3 includes major improvements for com
 - **WebRTC metrics**: Publish the standardized WebRTC metrics for all supported browsers. Deprecate `useBandwidthMetrics` hook in favor of `useMediaStreamMetrics` hook.
 - **Logger**: Add `LoggerProvider` and `useLogger` components to enable universal logging in component library.
 
-Check out our <a target="_blank" href="https://github.com/aws/amazon-chime-sdk-component-library-react/blob/2e9ae19fb6ea72a6611a7d7188744c1db45d8e09/CHANGELOG.md#L20">CHANGELOG for 3.0.0</a> for more details.
+Check out our [CHANGELOG for 3.0.0](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/2e9ae19fb6ea72a6611a7d7188744c1db45d8e09/CHANGELOG.md#L20) for more details.
```
Comment (xuesichao) [inline src/components/migrationToV3.stories.mdx:30]: Do we need to update other links in this PR?

File: src/components/migrationToV3.stories.mdx (hunk 1 above)
Comment (xuesichao) [reply src/components/migrationToV3.stories.mdx:542]: Thanks for catching this!

We could escape `|` in this way, but it's too complex and I prefer your solution.

| Before in 2.x                                              | After in 3.x                                                             |
| ---------------------------------------------------------- | ------------------------------------------------------------------------ |
| <code>selectedAudioOutputDevice: string &#124; null</code> | <code>selectedAudioOutputDevice: string &#124; null</code>               |
| <code>selectedAudioInputDevice: string &#124; null</code>  | <code>selectedAudioInputDevice: AudioInputDevice &#124; undefined</code> |
| <code>selectedVideoInputDevice: string &#124; null</code>  | <code>selectedVideoInputDevice: VideoInputDevice &#124; undefined</code> |

File: src/components/migrationToV3.stories.mdx (hunk 2 above)
Comment (devalevenkatesh) [reply src/components/migrationToV3.stories.mdx:30]: Nope, I thought you had a script which would easily update this 😅 thats why for test I made this change to know it works :). Happy to put another PR updating rest.

File: src/components/migrationToV3.stories.mdx (hunk 1 above)
Comment (devalevenkatesh) [reply src/components/migrationToV3.stories.mdx:542]: Cool dint knew about this! Well I kept it simple for now, can fix if this needs a revision :)

File: src/components/migrationToV3.stories.mdx (hunk 2 above)
Comment (xuesichao) [reply src/components/migrationToV3.stories.mdx:30]: Oh got it, let me raise a PR to update those, and I found several typos need to be fixed.

File: src/components/migrationToV3.stories.mdx (hunk 1 above)
Comment (xuesichao) [reply src/components/migrationToV3.stories.mdx:542]: No, it's good, no need to fix it

**Review by xuesichao — APPROVED** [review]

**Pull request merged**